L1Trigger/Phase2L1ParticleFlow/python/l1pfProducer_cfi.py

```diff
@@ -18,9 +19,9 @@
      puppiEtaCuts       = cms.vdouble(1.5, 2.5, 3.0, 5.5), 
      puppiPtCuts        = cms.vdouble(0.0, 3.0, 6.0, 8.0),
      puppiPtCutsPhotons = cms.vdouble(0.0, 3.0, 6.0, 8.0),
-     vtxRes      = cms.double(0.333),
+     vtxRes      = cms.double(0.15),
      vtxAlgo     = cms.string("TP"),
-     vtxAdaptiveCut = cms.bool(True),
+     vtxAdaptiveCut = cms.bool(False),
```

Comment:
for this change it would be good to see studies of what happens to the puppi jet response / resolution, the efficiency for associating tracks to vertices would be smaller.